changelog/unreleased/change-oc-breadcrumb-a11y

```diff
@@ -0,0 +1,8 @@
+Change: Extend accessibility to oc-breadcrumb component
```

Comment:
"Improve acccessibility of" or "Extend accessibility of", also make sure the headline isn't too long (otherwise CI fails 🤦🏽 )